components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -7823,6 +7823,37 @@ def setInvertedAxis(self, inverted):
         (1, False, True): 'b',  # signed char
         }
 
+    @assert_pixels
+    def getHistogram(self, channels, binCount, globalRange=True,
```

Comment:
I think we should either change all or none. Having two different styles in the same file will not be nice. I guess IF we're going to do it then 5.3.0 is as good a time as any. cc @chris-allan